WooCommerce/Classes/Notifications/PushNotificationsManager.swift

```diff
@@ -519,15 +531,27 @@ private extension PushNotificationsManager {
 // MARK: - PushNotification Extension
 
 private extension PushNotification {
-    static func from(userInfo: [AnyHashable: Any]) -> PushNotification? {
-        guard let noteID = userInfo.integer(forKey: APNSKey.identifier),
-              let message = userInfo.dictionary(forKey: APNSKey.aps)?.string(forKey: APNSKey.alert),
-              let type = userInfo.string(forKey: APNSKey.type),
-              let noteKind = Note.Kind(rawValue: type) else {
-            return nil
+    static func from(userInfo: [AnyHashable: Any], pushNotificationsForAllStoresEnabled: Bool) -> PushNotification? {
```

Comment:
Maybe a more apt argument name is `usingV2PayloadFormat`. 

```suggestion
    static func from(userInfo: [AnyHashable: Any], usingV2PayloadFormat: Bool) -> PushNotification? {
```

🤷

Reply:
This code is to be deleted after the multi-store push notifications release is stable, I thought it might be more obvious with the feature flag name? 😄